src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothAdapter.cs

```diff
@@ -371,6 +371,59 @@ static public int RemainingTimeAsVisible
             }
         }
 
+        /// <summary>
+        /// Enables the local Bluetooth adapter, asynchronously.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Enable()
+        {
+            int ret = BluetoothAdapterImpl.Instance.Enable();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to Enable - " + (BluetoothError)ret);
+                BluetoothErrorFactory.ThrowBluetoothException(ret);
+            }
+        }
+
+        /// <summary>
+        /// Disables the local Bluetooth adapter, asynchronously.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void Disable()
+        {
+            int ret = BluetoothAdapterImpl.Instance.Disable();
+            if (ret != (int)BluetoothError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to Disable - " + (BluetoothError)ret);
+                BluetoothErrorFactory.ThrowBluetoothException(ret);
+            }
+        }
+
+        /// <summary>
+        /// Sets the visibility mode.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">Thrown when the Bluetooth is not enabled.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public static void SetVisibility(VisibilityMode mode, int duration)
```

Comment:
Need description about duration

Reply:
Done. Thank you.

Reply:
`duration` it also apply with Discoverable mode?
and What mean `0` value of duration. 
And it ambiguous scenario

``` c#
SetVisibility(VisibilityMode.Discoverable, 10);
SetVisibility(VisibilityMode.TimeLimitedDiscoverable, 10);

// What happen with 0 duration and TimeLimitedDiscoverable?
SetVisibility(VisibilityMode.TimeLimitedDiscoverable, 0);
SetVisibility(VisibilityMode.Discoverable, 0);
```
Why need TimeLimitedDiscoverable value? we already have duration parameter.

Reply:
So, I recommend below interface
```
   SetVisibility(bool visible);
   SetVisibility(bool visible, int timeout);
```

Reply:
@myroot 
Thank you for your comment.
As you said, if VisibilityMode is not TimeLimitedDiscoverable, 'timeout' is no needed.
Then, how about below prototype?
SetVisibility(bool visible);
SetVisibility(int timeout); // because 'timeout' is existing, 'visible' is also no needed.